nodejs/packages/layer/src/wrapper.ts

```diff
@@ -504,21 +512,27 @@ async function initializeLoggerProvider(
   );
 
   const logExporter = new OTLPLogExporter();
-  const logRecordProcessors: LogRecordProcessor[] = [];
-  const loggerConfig = {
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  let loggerConfig: any = {
```

Comment:
Any reason why you would not use https://github.com/mfernd/opentelemetry-js/blob/main/experimental/packages/sdk-logs/src/index.ts#L18 here?